go/src/koding/klient/machine/dynamic.go

```diff
@@ -192,10 +192,14 @@ func (dc *DynamicClient) cron() {
 
 // tryUpdate uses client builder to ping the machine and updates dynamic client
 // if machine address changes.
-func (dc *DynamicClient) tryUpdate(addr *Addr) {
+func (dc *DynamicClient) tryUpdate(addr *Addr, islog *bool) {
 	stat, a, err := dc.opts.Builder.Ping(dc.opts.AddrFunc)
 	if err != nil {
-		dc.log.Warning("Machine ping error: %s", err)
+		if *islog {
```

Comment:
there is 26% chance of failure with pointer panic


>
<<<~spoiler~>>>
http://www.ign.com/videos/2016/12/12/rogue-one-a-star-wars-story-chance-of-failure-clip
<<<~/spoiler~>>>